File: advent-{{ cookiecutter.year }}/advent-typescript/src/day{{ cookiecutter.day }}.test.ts
import * as day from "./day{{ cookiecutter.day }}";
import { runTests } from "./utils/tests";

describe("part1", () => {
  runTests(1, 1, day.part1);
});

describe("part2", () => {
  runTests(1, 2, day.part2);
});

File: advent-{{ cookiecutter.year }}/advent-go/day{{ cookiecutter.day }}_test.go
package main

import (
	. "advent-go/utils"
	"testing"
)

func TestDay{{ cookiecutter.day }}Part1(t *testing.T) {
	RunTests(t, Day{{ cookiecutter.day }}Part1)
}

func TestDay{{ cookiecutter.day }}Part2(t *testing.T) {
	RunTests(t, Day{{ cookiecutter.day }}Part2)
}

func BenchmarkDay{{ cookiecutter.day }}Part1(b *testing.B) {
	RunBenchmarks(b, Day{{ cookiecutter.day }}Part1)
}

func BenchmarkDay{{ cookiecutter.day }}Part2(b *testing.B) {
	RunBenchmarks(b, Day{{ cookiecutter.day }}Part2)
}
